mandatory/2-function-creation.js

```diff
@@ -4,7 +4,14 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+function isAcceptableUser(userAge, isLoggedIn) {
+  const acceptableAge = 18;
+  if (userAge >= acceptableAge && isLoggedIn) {
```

Comment:
Nitpick: While this is OK, it's slightly better to do `isLoggedIn === true` as this type checks isLoggedIn to make sure it's passed into the function as a boolean.